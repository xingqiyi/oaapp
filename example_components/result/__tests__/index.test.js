import React from 'react';
import { render } from 'enzyme';
import { renderToJson } from 'enzyme-to-json';
import Result from '../index';

describe('Result', () => {
  it('renders correctly', () => {
    const wrapper = render(
      <Result
        imgUrl="https://zos.alipayobjects.com/rmsportal/qlMpMyJTcSjMpKAgtcEt.png"
        title="验证成功"
        message="所提交内容已成功完成验证"
      />,
    );
    expect(renderToJson(wrapper)).toMatchSnapshot();
    expect(wrapper.find('.am-result')).toHaveLength(1);
  });
});
