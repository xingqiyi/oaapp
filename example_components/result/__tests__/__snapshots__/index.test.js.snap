// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Result renders correctly 1`] = `
<div
  class="am-result"
  role="alert"
>
  <div
    class="am-result-pic"
    style="background-image:url(https://zos.alipayobjects.com/rmsportal/qlMpMyJTcSjMpKAgtcEt.png);"
  />
  <div
    class="am-result-title"
  >
    验证成功
  </div>
  <div
    class="am-result-message"
  >
    所提交内容已成功完成验证
  </div>
</div>
`;
